twitter.html

```diff
@@ -0,0 +1,349 @@
+    <title>Twitter Clone</title>
+    <script src="https://kit.fontawesome.com/ecdbe10a83.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="twitter.css">
+    <link rel="icon" type="image/twitter-icon" href="./assets/icons/icons8-twitter-48.png">
+</head>
+<body>
+    <div class="header">
+        <div class="left-header">
+            <a href="./twitter.html" class="logo"><i class="fa-brands fa-twitter"></i></a>
+        </div>
+        <div class="main-header a">
+            <a href="#" class="home"><h3>Home</h3></a>
+            <a href="#"><i class="fa-solid fa-timeline" style="color: #00acee;"></i>
+            </a>
+        </div>
+        <div class="right-header">
+            &nbsp
+            <i class="fa-solid fa-magnifying-glass" style="color: #717274;"></i>
+            &nbsp
+            <input type="search" name="search-twitter" id="searchbar" placeholder="Search Twitter">
+        </div>
+    </div>
+    <div class="container">
+        <div class="left-container">
+            <div class="left-top">
+                <div class="left-container-logo">
+                    <ul>
+                        <li>
+                            <a href="#">
+                                <i class="fa-solid fa-house-user"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <i class="fa-solid fa-hashtag"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <i class="fa-regular fa-bell"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <i class="fa-regular fa-envelope"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <i class="fa-regular fa-bookmark"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <i class="fa-regular fa-rectangle-list"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <i class="fa-regular fa-user"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <i class="fa-solid fa-ellipsis"></i>
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="left-container-name">
+                    <ul>
+                        <li><a href="#">Home</a></li>
```

Comment:
Was it easier to target the listed item's in css instead of using divs? just curious that's all.